api/server/router/container/copy.go

```diff
@@ -103,7 +105,9 @@ func writeCompressedResponse(w http.ResponseWriter, r *http.Request, body io.Rea
 	default:
 		cw = w
 	}
-	_, err := io.Copy(cw, body)
+	output := ioutils.NewWriteFlusher(cw)
```

Comment:
This doesn't work. The compressed writer is not a flusher.

Reply:
:confused:

Reply:
I pushed an update that fixes this, adds a test, and a max buffer for authz responses.